### DISCUSSION TITLE
最近TCP模式下载测速异常为0，这是什么原因造成的？

### DISCUSSION THREAD

**Discussion by TranscendMa** [XIU2/CloudflareSpeedTest#560] [提问]
之前一直用的默认参数。在10月初的时候能够正常使用，默认参数每次测速都可以测试到一个较低的延迟和>30MB/s的速度，直到11月底左右进行测速时，速度最快降低到只有2~3MB/s。最近两周测速情况变为完全无下载速度、但是延迟测速正常。
结果类似于
[IP 地址           已发送  已接收  丢包率  平均延迟  下载速度 (MB/s)
104.17.176.195    4       4       0.00    40.53     0.00
104.17.220.216    4       4       0.00    40.56     0.00]

阅读readme以后，测试发现，若使用HTTP模式则测速结果正常。ISP是江苏移动。

请问是什么导致了这样的情况？如果是ISP导致的，如何防止情况进一步恶化？
另外，Cloudflare于2024 年 12 月 3 日更新使用协议，禁止将其服务用作代理用途，请问会不会对用Cloudflare CDN加速访问造成影响？

**Comment by XIU2** [comment]
1. 尝试 `-url` 指定其他下载测速地址试试
2. 不清楚，因为我不用套 CDN 的代理
